src/Symfony/Bridge/Monolog/Command/ServerLogCommand.php

```diff
@@ -156,6 +158,16 @@ private function displayLog(OutputInterface $output, int $clientId, array $recor
         $logBlock = sprintf('<bg=%s> </>', self::BG_COLOR[$clientId % 8]);
         $output->write($logBlock);
 
+        if (Logger::API >= 3) {
+            $record = new LogRecord(
+                level: Level::fromValue($record['level']),
+                channel: $record['channel'],
+                context: $record['context']->getContext(),
+                datetime: $record['datetime'],
+                message: $record['message'],
```

Comment:
let's not use named arguments if possible

Reply:
Ok I did the change. But why don't use named arguments ?
Named arguments are already used for the LogRecord. (https://github.com/symfony/symfony/blob/547485e99c2ce377211679674f18c90ccc7fb753/src/Symfony/Bridge/Monolog/Tests/RecordFactory.php#L25)